### DISCUSSION TITLE
How to migrate the database

### DISCUSSION THREAD

**Discussion by happy-bits** [jbogard/ContosoUniversityDotNetCore-Pages#68] [Q&A]
Hi!

Thanks for a great project 😊 This is very useful to understand CQRS, Vertical slice architecture and more!

If I want to extend the model/database (e.g add a property/column to Student), what is the preferred way of doing that? I tried to do

`Add-Migration X1`

...but that created a script for the whole database. Should I instead use psake in some way?

Regards
Oscar

**Comment by happy-bits** [comment]
Okay, I solved it by 

1) Changing the connectionstring to point to a new database (non existing)
2) Add-Migration X1
3) Update-Database
4) Run 0002_InitialData.sql

Please let me know if there are a better way 😊

**Reply by jbogard** [reply]
What is `Add-Migration`? I'm not familiar with that command.

But in this scenario, because it's using `grate`, it's up to you to create your migration scripts, however you like.
